packages/wrangler/src/config/index.ts

```diff
@@ -29,7 +29,7 @@ export type {
 export function readConfig<CommandArgs>(
 	configPath: string | undefined,
 	// Include command specific args as well as the wrangler global flags
-	args: CommandArgs & OnlyCamelCase<CommonYargsOptions>
+	args: CommandArgs & Partial<OnlyCamelCase<CommonYargsOptions>>
```

Comment:
Why this change?

Reply:
Because otherwise I would have to specify extra/unused properties here: https://github.com/cloudflare/workers-sdk/blob/693f00cbe89b6adb818be367d0b72487af79b505/packages/wrangler/src/api/integrations/bindings/index.ts#L75-L77

But actually thanks for pointing this out, I've replaced this with
```ts
Pick<OnlyCamelCase<CommonYargsOptions>, "experimentalJsonConfig">
```
since `experimentalJsonConfig` seems to be the only flag used in this function, although I am not sure if this is totally correct, since later `args` gets passed here:
https://github.com/cloudflare/workers-sdk/blob/693f00cbe89b6adb818be367d0b72487af79b505/packages/wrangler/src/config/index.ts#L46-L50

(but the `args` parameter in `normalizeAndValidateConfig` is an `unknown`.... so maybe it's ok? 😕)

Reply:
Yeah that's fine.